client/src/main/java/org/asynchttpclient/DefaultAsyncHttpClient.java

```diff
@@ -112,6 +115,15 @@ public void close() {
                     LOGGER.warn("Unexpected error on HashedWheelTimer close", t);
                 }
             }
+            
+            //see https://github.com/netty/netty/issues/2084#issuecomment-44822314
+            try {
+                ThreadDeathWatcher.awaitInactivity(5, TimeUnit.SECONDS);
```

Comment:
Shouldn't the await activity be aligned with `AsyncHttpClientConfig#getShutdownTimeout` or `AsyncHttpClientConfig#getShutdownQuietPeriod`?

See https://github.com/AsyncHttpClient/async-http-client/blob/master/client/src/main/java/org/asynchttpclient/netty/channel/ChannelManager.java#L309